sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -184,8 +184,8 @@ object SQLConf {
   val OPTIMIZER_PLAN_CHANGE_LOG_LEVEL = buildConf("spark.sql.optimizer.planChangeLog.level")
     .internal()
     .doc("Configures the log level for logging the change from the original plan to the new " +
-      "plan after a rule is applied. The value can be 'trace', 'debug', 'info', 'warn', or " +
-      "'error'. The default log level is 'trace'.")
+      "plan after a rule or batch is applied. The value can be 'trace', 'debug', 'info', " +
```

Comment:
Is this config relate to the batch?

Reply:
@maropu  yes, this config is common for both rule and batch(to set a specific log level for logging plan changes after a rule or batch is applied)

Reply:
Yes, @maropu. This "level" is used both by batch and rule logging. The other conf you pointed out below is for "which batches to log".